package.json

```diff
@@ -6,29 +6,34 @@
         "node": "^18.18.0 || ^20.9.0 || >=21.1.0"
     },
     "main": "lib/index.js",
+    "types": "types/index.d.ts",
     "files": [
         "lib/",
-        "configs/"
+        "configs/",
+        "types/index.d.ts"
     ],
     "peerDependencies": {
         "eslint": ">=8.23.0"
     },
     "dependencies": {
         "@eslint-community/eslint-utils": "^4.4.0",
+        "@types/eslint": "^8.56.2",
```

Comment:
can we remove it here - it's already listed in `devDependencies`?

Reply:
Unfortunatly we can't until [#213](https://github.com/eslint-community/eslint-plugin-n/issues/213) is done as the types require the dependency :)